contracts/common/Enum.sol

```diff
@@ -6,7 +6,6 @@ pragma solidity ^0.5.0;
 contract Enum {
     enum Operation {
         Call,
-        DelegateCall,
-        Create
```

Comment:
We are not allowing to create contracts from the Safe anymore?

Reply:
You can use a deletegatecall to the new `CreateCall` lib to create contracts. This will also enforce that for creation not more than `safeTxGas` is used.